.storybook/preview.jsx

```diff
@@ -1,5 +1,6 @@
 import React, {StrictMode} from 'react'
 import { EbaySvg } from '../src/ebay-svg'
+import '@storybook/addon-console';
```

Comment:
Lets not add this one for now. We use `actions` tabs for showing the callbacks and this plugin also uses it, having this might be confusing. (we can miss let log anyway if you don't open this tab)

Reply:
> we can miss the log anyway if you don't open this tab

Sure, but having your dev console open is a pretty standard part of development. Is there a way to see the error logs without this?

Regardless, removing per your request.

Reply:
It shows in the normal console on local. It doesn't show in the deployed version because it is a prod build, so warnings are removed by React